praas-app-ionic/src/components/Conduit/Form/index.tsx

```diff
@@ -22,6 +22,19 @@ type Props = {
   onSave: (conduit: any) => void;
 };
 
+const prefixList: { [key: string]: string[] } = {
+  Airtable: ['https://api.airtable.com/v0/'],
+  'Google Sheets': [
```

Comment:
could this be replaced with a possible more readable mapping in the FE? 
Eg: const SERVICES =  {
'Airtable': 'airtable',
'Google Sheets': 'google_sheets'
}
This would allow a middle mapping which would allow the code to be cleaner